Esmorga-Backend/esmorga-backend
# [MOB-25] ENV Vars Validations

https://mobilemakers.atlassian.net/browse/MOB-25

.env
```
MONGODB_URI = mongodb://localhost:27017/esmorga
REDIS_HOST = localhost
REDIS_PORT = 6379
APP_PORT = 3000
```

**Testing guide:**
- Start up the app and check all dependencies initialized
- Execute a flow to check everything continue working correctly. Example: 
```
curl --location 'http://localhost:3000/v1/events'
```
